gqlalchemy/query_builder.py

```diff
@@ -573,13 +586,27 @@ def to(
             raise InvalidMatchChainException()
 
         if relationship is None:
-            type_str = to_cypher_labels(edge_label)
+            labels_str = to_cypher_labels(edge_label)
```

Comment:
Okay, I guess this is related to https://github.com/memgraph/gqlalchemy/issues/123
I would leave `type_str` because of this, edge does not have a label, but type and I think we should change that.

Reply:
Great comment! I put this before the discussions, because the `to()` argument is named `edge_label`, and more importantly, `EdgePartialQuery` accepts `labels``(=labels_str). And also, `from_()` method right below `to()` uses labels_str, so a quick fix seemed appropriate. 
I will revert this and leave my comments on the issue, so we can adjust that separatly.